app/components/primer/octicon_component.rb

```diff
@@ -35,8 +37,8 @@ class OcticonComponent < Primer::Component
     def initialize(icon_name = nil, icon: nil, size: SIZE_DEFAULT, use_symbol: false, **system_arguments)
       icon_key = icon_name || icon
 
-      # Don't allow sizes under 16px
-      if system_arguments[:height].present? && system_arguments[:height].to_i < 16 || system_arguments[:width].present? && system_arguments[:width].to_i < 16
+      # Don't allow sizes under 12px
+      if system_arguments[:height].present? && system_arguments[:height].to_i < 12 || system_arguments[:width].present? && system_arguments[:width].to_i < 12
```

Comment:
Let's not change this, but allow the xsmall size argument. Ideally I would like to drop support for height and width completely. but that's scoped in another issue.